specifyweb/frontend/js_src/lib/components/FormParse/__tests__/cells.test.ts

```diff
@@ -129,7 +131,7 @@ describe('parseFormCell', () => {
       parseFormCell(
         schema.models.CollectionObject,
         strictParseXml(
-          '<cell invisible="true" type=" test2 " initialize="align=Center" colSpan=" 5 " id="test" />'
+          '<cell invisible="true" type=" test2 " initialize="align=center; verticalAlign:center" colSpan=" 5 " id="test" />'
```

Comment:
> verticalAlign:center

This doesn't seem to be consistent with other attributes defined in a cell's `initialize` (it's also inconsistent with other verticalAlign (sub?)attributes used in this file)

Should it be `verticalAlign=center`?

So the initialize attribute would look like: 
```xml
initialize="align=center; verticalAlign=center"
```